.github/workflows/linux.yml

```diff
@@ -279,7 +279,7 @@ jobs:
         shell: bash
     runs-on: ubuntu-20.04
```

Comment:
looks like Azure Container Registry is not accessible from default github runners

Reply:
Why not use the aks-4-core runners? They have access to those images + they seem to be cheaper than even 2-core GHA: https://azureprice.net/vm/Standard_F4s vs. https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#per-minute-rates

Reply:
I suppose having 2 cores aks runners